tests/memex/search/index_test.py

```diff
@@ -93,43 +93,100 @@ def log(self, patch):
         return patch('memex.search.index.log')
 
 
-@pytest.mark.usefixtures('BatchIndexer', 'BatchDeleter')
+@pytest.mark.usefixtures('BatchDeleter', 'BatchIndexer', 'configure_index')
 class TestReindex(object):
-    def test_it_indexes_all_annotations(self, BatchIndexer):
-        index.reindex(mock.sentinel.session, mock.sentinel.es, mock.sentinel.request)
+    def test_indexes_annotations(self, es, indexer):
```

Comment:
I don't really see a reason to change the function name from "test_it_..." to "test_...", but that would be nicer in a separate commit for easier reviewing.